pandas/core/arrays/datetimes.py

```diff
@@ -1967,12 +1967,12 @@ def sequence_to_datetimes(
 
 def sequence_to_dt64ns(
     data,
-    dtype=None,
-    copy=False,
-    tz=None,
-    dayfirst=False,
-    yearfirst=False,
-    ambiguous="raise",
+    dtype: str = None,
```

Comment:
probably should be `Dtype | None`

Reply:
Ah, agreed - Dtype is better. For optional keyword arguments, isn't the `| None` implied?

Reply:
nope

https://github.com/pandas-dev/pandas/blob/3bb31da431ee4b06df8e37666dacbc299e993a55/setup.cfg#L213

Reply:
Thanks Marco - added this in the latest!